src/remediations/controller.write.js

```diff
@@ -170,7 +170,24 @@ exports.patch = errors.async(async function (req, res) {
         }
 
         if (name) {
+            const existing_remediation = await db.remediation.findOne({
+                attributes: ['id'],
+                where: { created_by: username, name: name },
+            }, {
+                transaction
+            });
+
+            // Do not allow users to use the same name for more than one remediation
+            if (existing_remediation) {
+                throw new errors.BadRequest('DUPLICATE_NAME', `Remediation name "${name}" for user "${username}" has already been taken`);
```

Comment:
Does it need to inform about the username? And does it even have to be tied to a specific user in an org?

Reply:
My understanding was that we don't want to allow remediations to have the same name "on an account basis" so basically a user shouldn't be able to create a remediation or update a remediation using the same name as an existing one under their account.

Reply:
Hmm good point.. Yeah, I'm free to talk about this today or we can do tomorrow.